commit 83ba254688f0edd6fa29512e538c721f1f46a424
Author: Andrew Morton <akpm@osdl.org>
Date:   Fri Dec 8 02:39:49 2006 -0800

    [PATCH] fault-injection: stacktrace filtering kconfig fix
    
    `select' doesn't work very well.  With alpha `make allmodconfig' we end up
    with CONFIG_STACKTRACE enabled, so we end up with undefined save_stacktrace()
    at link time.
    
    Cc: Akinobu Mita <akinobu.mita@gmail.com>
    Cc: Don Mullis <dwm@meer.net>
    Signed-off-by: Andrew Morton <akpm@osdl.org>
    Signed-off-by: Linus Torvalds <torvalds@osdl.org>

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index c24db7221b55..6eccc643c32e 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -417,7 +417,7 @@ config LKDTM
 config FAULT_INJECTION
 	bool "Fault-injection framework"
 	depends on DEBUG_KERNEL
-	select STACKTRACE
+	depends on STACKTRACE
 	select FRAME_POINTER
 	help
 	  Provide fault-injection framework.